redux-1/src/LocalStorageManager.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+
+import { getAttendingInformation } from './selectors';
+import { changeAttendingInformation } from './actions';
+
+import type { Attending } from './types/state';
+
+type Props = {|
+  +attendingInformation: Attending[],
+  +changeAttendingInformation: typeof changeAttendingInformation,
+|};
+
+class LocalStorageManager extends PureComponent<Props> {
+  componentDidMount() {
+    const { changeAttendingInformation } = this.props;
+
+    try {
+      const storedAttending = JSON.parse(
+        window.localStorage.getItem('attending')
+      );
+      if (storedAttending) {
+        changeAttendingInformation(storedAttending);
+      }
+    } catch (err) {
+      console.error('Error while parsing stored `attending` data.');
+    }
+  }
+
+  componentDidUpdate() {
```

Comment:
Is the unnecessary localstorage update when hydrating from existing localstorage intended as an improvement for the next version?

Reply:
je l'avais identifié et je me suis dit que l'impact en terme de perf allait être mineur, et le corriger allait forcément apporter de la complexité que je trouvais un peu inutile.
donc je l'ai laissé pur plus tard si on avait envie, mais perso je pense qu'il faut pas le changer.